### PR TITLE
Set bbb-origin and bbb-origin-server-name (#87)

### LINKED ISSUE #87: FR: Set bbb-origin and bbb-origin-server-name
In order to identify meetings started by the Mattermost plugin, please add the origin metadata when creating the meeting. See Greenlight for reference.

### PR DESCRIPTION
**Pull request by moschlar**
Closes #87.

Used https://github.com/bigbluebutton/greenlight/blob/master/app/controllers/concerns/bbb_server.rb#L72-L74 as reference

### CODE REVIEW

File: server/bigbluebuttonapiwrapper/api/APIcalls.go [hunk 1]
```diff
@@ -80,9 +81,17 @@ func CreateMeeting(meetingRoom *dataStructs.MeetingRoom) (string, error) {
 		url.QueryEscape(meetingRoom.Meta_endcallbackurl)
 	voiceBridge := "&voiceBridge=" + url.QueryEscape(meetingRoom.VoiceBridge)
 
+	metaBBBOrigin := "&meta_bbb-origin=bbb-mm" +
+		"&meta_bbb-origin-version=" + url.QueryEscape(helpers.PluginVersion)
+
+	hostname, err := os.Hostname()
```
Comment (rottaran) [inline server/bigbluebuttonapiwrapper/api/APIcalls.go:87]: The operating system's hostname is not always unique. For example with containters or multiple mattermost instances on the same server. Wouldn't it be more reliable to take the serverURL from the mattermost configuration?

Comment (moschlar) [reply server/bigbluebuttonapiwrapper/api/APIcalls.go]: Absolutely!

**Comment by moschlar** [comment]
Ping?

**Review by harshilsharma63 — APPROVED** [review]
LGTM

**Pull request merged**